src/pages/payment-methods/_mobile-expanded-list.tsx

```diff
@@ -297,41 +297,41 @@ const MobileExpandedList = ({
-                            </Header>
-                        </StyledItemDiv>
+                    {(payment_data?.reference || payment_data?.reference_link) && (
+                        <StyledRow jc="space-between" ai="center">
+                            <StyledItemDiv>
+                                <Header as="p" type="subtitle-2">
+                                    {is_dp2p ? localize('More info') : localize('Reference')}
+                                </Header>
+                            </StyledItemDiv>
 
-                        <StyledKeyDiv>
-                            <>
-                                {payment_data.reference ? (
-                                    <RefIcon
-                                        href={`/payment-methods/${
-                                            payment_data.locales?.includes(locale?.locale?.language)
-                                                ? locale?.locale?.language +
-                                                  '/' +
-                                                  payment_data.reference
-                                                : payment_data.reference
-                                        }`}
-                                        target="_blank"
-                                        rel="noopener noreferrer"
-                                    >
-                                        <StyledPDF src={PDF} alt="PDF" />
-                                    </RefIcon>
-                                ) : payment_data.reference_link ? (
-                                    <StyledRefLink jc="flex-end">
-                                        {payment_data.reference_link}
-                                    </StyledRefLink>
-                                ) : (
-                                    <ValueText is_rtl={is_rtl} type="subtitle-2" weight="normal">
-                                        -
-                                    </ValueText>
-                                )}
-                            </>
-                        </StyledKeyDiv>
-                    </StyledRow>
+                            <StyledKeyDiv>
+                                <>
+                                    {payment_data.reference ? (
+                                        <RefIcon
+                                            href={`/payment-methods/${
+                                                payment_data.locales?.includes(
+                                                    locale?.locale?.language,
+                                                )
+                                                    ? locale?.locale?.language +
+                                                      '/' +
+                                                      payment_data.reference
+                                                    : payment_data.reference
+                                            }`}
```

Comment:
Your solution will work, but we have to spent some time to realize what happens here. Better to decrease cognitive complexity

```suggestion
     const language = getLanguage()
     
     href={language == 'en' ? 
     `/payment-methods/${payment_data.reference}` :
     `/payment-methods/${language}/{payment_data.reference}
     `}
```

Reply:
Hi @NikitK-deriv, I think the logic here is a little bit different from your suggestion. It's not checking language from the param but from props

Reply:
I see, we have to keep it while we have no all translated pdf's yet 👍